scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn-24.yaml

```diff
@@ -177,6 +177,70 @@ QFN-24-1EP_3x4mm_P0.4mm_EP1.65x2.65mm:
   #suffix: '_Pad{pad_x:.2f}x{pad_y:.2f}mm_HandSolder'
   #include_suffix_in_3dpath: 'False'
 
+QFN-24-1EP_4x4mm_P0.5mm_EP2.15x2.15mm:
+  device_type: 'QFN'
+  #manufacturer: 'man'
+  #part_number: 'mpn'
+  size_source: 'http://ww1.microchip.com/downloads/en/PackagingSpec/00000049BQ.pdf#page=278'
+  ipc_class: 'qfn' # 'qfn_pull_back'
+  #ipc_density: 'least' #overwrite global value for this device.
+  # custom_name_format:
+  body_size_x:
+    nominal: 4
+    tolerance: 0.15
+  body_size_y:
+    nominal: 4
+    tolerance: 0.15
+  overall_height:
+    minimum: 0.8
+    nominal: 0.90
+    maximum: 1
+
+  lead_width:
+    minimum: 0.18
+    nominal: 0.25
+    maximum: 0.3
+  lead_len:
+    minimum: 0.3
+    nominal: 0.4
+    maximum: 0.5
+
+  # size choosen for backwards compatibility
+  EP_size_x:
+    minimum: 2.0
+    nominal: 2.15
+    maximum: 2.25
+  EP_size_x_overwrite: 2.15
+  EP_size_y:
+    minimum: 2.0
+    nominal: 2.15
+    maximum: 2.25
+  EP_size_y_overwrite: 2.15
+  # EP_paste_coverage: 0.65
+  EP_num_paste_pads: [2, 2]
+
+  thermal_vias:
+    count: [3, 3]
+    drill: 0.2
```

Comment:
Please increase to 0.3 for cheaper manufacturability.

Reply:
Thank you for the review, just committed changes